build/common.targets

```diff
@@ -46,8 +46,13 @@
     <Import Project="test.targets" />
   </ImportGroup>
 
+  <!-- Centrally managed packages -->
+  <ImportGroup Condition=" '$(SkipCentralPackageVersions)' != 'true' ">
+    <Import Project="packages.targets" />
+  </ImportGroup>
```

Comment:
At first I asked what's the point of the condition, and while I don't understand why the version conflict is not resolvable, I think it'd be worthwhile adding a comment to explain why the condition exists.

Reply:
fair, I'll add a comment.

As far as the condition goes, it's there to allow me to enforce no one is managing the version numbers inside the project files.

> and while I don't understand why the version conflict is not resolvable

Package versions are missing :( 
2 projects have 3 packages, 2 each, one A, B, the other B, C.
The 2 projects share the B, which has `=` dependencies, but the same versions of A and C are nowhere to be found.